File: Serene/Serene.App/Modules/Administration/Language/LanguageForm.cs

namespace Serene.Administration.Forms
{
    using Serenity.ComponentModel;
    using System;

    [FormScript("Administration.Language")]
    [BasedOnRow(typeof(Entities.LanguageRow))]
    public class LanguageForm
    {
        public String LanguageId { get; set; }
        public String LanguageName { get; set; }
    }
}

File: src/Serene.Web/Modules/ServerTypes/Administration/LanguageForm.ts
import { StringEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface LanguageForm {
    LanguageId: StringEditor;
    LanguageName: StringEditor;
}

export class LanguageForm extends PrefixedContext {
    static formKey = 'Administration.Language';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!LanguageForm.init)  {
            LanguageForm.init = true;

            var w0 = StringEditor;

            initFormType(LanguageForm, [
                'LanguageId', w0,
                'LanguageName', w0
            ]);
        }
    }
}
